docs/openrpc.json

```diff
@@ -0,0 +1,917 @@
+          },
+          "changeAddress": {
+            "description": "When action type is 'deposit', change addess is optional and specifies the destination for the change output after selecting UTXOs. When action type is 'withdrawal', it is ignored.",
+            "$ref": "#/components/schemas/Address"
+          }
+        }
+      },
+      "TransactionHex": {
+        "description": "Transaction encoded in hexadecimal.",
+        "type": "string"
+      },
+      "NanoContract": {
+        "type": "object",
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/Transaction"
+          },
+          {
+            "properties": {
+              "id": {
+                "type": "string"
+              },
+              "method": {
+                "type": "string"
+              },
+              "args": {
+                "type": "array",
+                "items": {
+                  "$ref": "#/components/schemas/Buffer"
+                }
+              },
+              "pubkey": {
+                "$ref": "#/components/schemas/Buffer"
+              },
+              "signature": {
+                "$ref": "#/components/schemas/Buffer"
+              }
+            }
+          }
+        ]
+      },
+      "Transaction": {
+        "type": "object",
+        "properties": {
+          "inputs": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/Input"
+            }
+          },
+          "outputs": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/Output"
+            }
+          },
+          "signalBits": {
+            "type": "number"
+          },
+          "version": {
+            "type": "number"
+          },
+          "weight": {
+            "type": "number"
+          },
+          "nonce": {
+            "type": "number"
+          },
+          "timestamp": {
+            "type": "number"
+          },
+          "parents": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          },
+          "tokens": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/Token"
+            }
+          },
+          "hash": {
+            "type": "string"
+          }
+        }
+      },
+      "Input": {
+        "type": "object",
+        "properties": {
+          "hash": {
+            "title": "Hash of transaction being spent",
+            "type": "string"
+          },
+          "index": {
+            "title": "Index of outputs array from output being spent",
+            "type": "number"
+          },
+          "data": {
+            "title": "Input signed data for P2PKH and redeemScript for P2SH",
+            "$ref": "#/components/schemas/Buffer"
+          }
+        }
+      },
+      "Output": {
+        "type": "object",
+        "properties": {
```

Comment:
Does it have timelock?

Reply:
Suggestion: keep as is:
- An output only has:
- value
- token_data
- script
- Timelock is a property within the script.
- However, script here is encoded as base64, the script properties do not appear.